index.js

```diff
@@ -25,6 +25,7 @@ module.exports = declare((api, options) => {
     removePropTypes,
     looseClasses = false,
     runtimeVersion,
+    useESModules = false,
```

Comment:
I think as you have written it, this is a breaking change. But I think we could downgrade it to a minor update if the default value for this option was based on the modules option, with a way to override it by passing it in.